### PR TITLE
Add icon titles and remove actions in process list

### LINKED ISSUE #167: Add title on submission status icons to help understand their meaning

### LINKED ISSUE #168: Maybe change the icon on the big "actions" button of the submission list

### LINKED ISSUE #182: Add a "back to submissions" link to each submission step
## Purpose

It appears that this link will be useful to quickly make a new submission.

## Proposal

Add a "back to submissions" link similar to the submission detail template.

### PR DESCRIPTION
**Pull request by willdurand**
Fix #167 
Fix #168
Fix #182

---

screen shot 2018-01-30 at 15 23 21
screen shot 2018-01-30 at 15 23 17
screen shot 2018-01-30 at 15 23 12

### CODE REVIEW

File: assets/scss/_submission.scss [hunk 1]
```diff
@@ -113,8 +113,7 @@
             text-align: center;
           }
 
-          &:nth-child(4),
-          &:last-child {
+          &:nth-child(4) {
```
Comment (jmaupetit) [inline assets/scss/_submission.scss]: Can you make the last child (last column) also right aligned please?

File: apps/submission/templates/submission/submission/process_detail.html [hunk 2]
```diff
@@ -8,6 +8,10 @@
 
 {% block header %}
 <h1>
+  <a href="{% url 'submission:index' %}">
+    Submissions
+  </a>
+  /
```
Comment (jmaupetit) [inline apps/submission/templates/submission/submission/process_detail.html]: This is great!

File: assets/scss/_submission.scss (hunk 1 above)
Comment (willdurand) [reply assets/scss/_submission.scss]: the status you mean? it is not really pretty when right aligned and the status was centered before.

Comment (willdurand) [reply assets/scss/_submission.scss]: the status you mean? it is not really pretty when right aligned and the status was centered before.

Comment (jmaupetit) [reply assets/scss/_submission.scss]: Yes, but the status wasn't the last column 😉 But ok then, it's a matter of taste.

Comment (willdurand) [reply assets/scss/_submission.scss]: I see. Let's try something.

Comment (willdurand) [reply assets/scss/_submission.scss]: how about this?

screen shot 2018-01-30 at 16 21 03

Comment (jmaupetit) [reply assets/scss/_submission.scss]: 👍

File: assets/scss/_submission.scss [hunk 3]
```diff
@@ -137,16 +136,11 @@
...
-          }
-
-          .actions {
-            width: 15%;
             text-align: right;
 
-            a.action {
-              margin-bottom: 0;
+            i {
+              margin-right: 13px;
```
Comment (jmaupetit) [inline assets/scss/_submission.scss]: Wow that's pixel-perfect! 😜

**Pull request merged**